files/homeassistant/hacs_version/individual_control.yaml

```diff
@@ -31,7 +31,7 @@ cards:
             - value: 'off'
               icon: mdi:power
               color: grey
-          aspect_ratio: 2
+          aspect_ratio: 1
```

Comment:
I think since we have some consistency between how this renders on our screens, we should keep this as 2 for now and look at it later.

Reply:
Agree, I've made the change also in my branch so I don't keep pushing with the 1 value 😉